vscode/src/extension.ts

```diff
@@ -238,3 +240,18 @@ export class QsTextDocumentContentProvider
     return getLibrarySourceContent(uri.path);
   }
 }
+
+function checkForOldQdk() {
+  const extension = vscode.extensions.getExtension(
+    "quantum.quantum-devkit-vscode"
+  );
+
+  const warningMessage =
+    "Previous QDK Extension `quantum.quantum-devkit-vscode` found. We recommend uninstalling the prior QDK before using this release.";
```

Comment:
Is any user going to know what `quantum.quantum-devkit-vscode` is? Can we add something more friendly like "Two Q# extensions are currently active - the new Azure Quantum Development Kit Preview, and the prior Microsoft Quantum Development Kit. Please disable or remove one of them to avoid conflicts".

Also, we discussed detected if they have both the 'dev' channel and 'stable' channel of the new extension installed and warning on that also.